packages/cyberstorm/src/components/Layout/PackageDetailLayout/PackageDetailLayout.tsx

```diff
@@ -90,26 +85,28 @@ export function PackageDetailLayout(props: PackageDetailLayoutProps) {
   }
   if (packageData.gitHubLink) {
     packageDetailsMeta.push(
-      <Link
-        key="2"
-        externalUrl={packageData.gitHubLink}
-        label="GitHub"
-        leftIcon={<FontAwesomeIcon icon={faGithub} fixedWidth />}
-      />
+      <a key="2" href={packageData.gitHubLink}>
```

Comment:
Buttons aren't supposed to be nested inside anchors. We probably want to keep some sort of `Link` element for links leading outside the website.

Reply:
PlainButton is not a <button>, but a <div>🤔  There _was_ a Link component, but that was only used in 2 places, both of which looked exactly like buttons